app/presenters/idv/max_attempts_failure_presenter.rb

```diff
@@ -32,7 +32,7 @@ def message
     end
 
     def next_steps
-      [sp_step, help_step, profile_step].compact
```

Comment:
We should probably go ahead and clean up these methods since they're dead code, right?